src/python/pants/backend/go/util_rules/import_analysis.py

```diff
@@ -39,7 +39,8 @@ async def determine_go_std_lib_imports() -> GoStdLibImports:
     list_result = await Get(
         ProcessResult,
         GoSdkProcess(
-            command=("list", "-json", "std"),
+            # "-find" skips determining dependencies and imports for each package.
+            command=("list", "-find", "-json", "std"),
```

Comment:
Does `-find` still list the `Target` field in this case? I.e., the package archives for stdlib.

Reply:
Yes:

```
$ go list -find -json std
{
	"Dir": "/opt/homebrew/Cellar/go/1.17/libexec/src/vendor/golang.org/x/text/unicode/norm",
	"ImportPath": "vendor/golang.org/x/text/unicode/norm",
	"Name": "norm",
	"Doc": "Package norm contains types and functions for normalizing Unicode strings.",
	"Target": "/opt/homebrew/Cellar/go/1.17/libexec/pkg/darwin_arm64/vendor/golang.org/x/text/unicode/norm.a",
	"Root": "/opt/homebrew/Cellar/go/1.17/libexec",
	"Match": [
		"std"
	],
	"Goroot": true,
	"Standard": true,
	"Stale": true,
	"StaleReason": "build ID mismatch",
	"GoFiles": [
		"composition.go",
		"forminfo.go",
		"input.go",
		"iter.go",
		"normalize.go",
		"readwriter.go",
		"tables13.0.0.go",
		"transform.go",
		"trie.go"
	],
	"IgnoredGoFiles": [
		"tables10.0.0.go",
		"tables11.0.0.go",
		"tables12.0.0.go",
		"tables9.0.0.go"
	]
}
```